C:37:"PHPUnit\Runner\DefaultTestResultCache":4594:{a:2:{s:7:"defects";a:13:{s:90:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetCompiledDeck with data set "empty data"";i:3;s:91:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetCompiledDeck with data set "stored data"";i:3;s:83:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testInitDeck with data set "empty data"";i:3;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testInitDeck with data set "stored data"";i:3;s:61:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeck";i:5;s:94:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page -1"";i:4;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page 0"";i:4;s:98:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page string"";i:4;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page 3"";i:4;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCard with data set "success 2"";i:4;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 1"";i:3;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCard with data set "success 1"";i:4;s:7:"Warning";i:6;}s:5:"times";a:35:{s:39:"KompiTech\Tests\App\EmptyTest::testTest";d:0.157;s:34:"Tests\CardsApp\EmptyTest::testTest";d:0.005;s:38:"Tests\CardsAppTest\EmptyTest::testTest";d:0.005;s:90:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetCompiledDeck with data set "empty data"";d:0.008;s:91:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetCompiledDeck with data set "stored data"";d:0;s:83:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testInitDeck with data set "empty data"";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testInitDeck with data set "stored data"";d:0;s:61:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeck";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeck with data set "page 1"";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeck with data set "page 2"";d:0;s:94:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page -1"";d:0.001;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page 0"";d:0;s:98:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page string"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testGetListedDeckException with data set "page 3"";d:0;s:87:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCard with data set "success 1"";d:0;s:87:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCard with data set "success 2"";d:0;s:96:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCardException with data set "failure 1"";d:0;s:96:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCardException with data set "failure 2"";d:0;s:96:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCardException with data set "failure 3"";d:0;s:96:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testCreateNewCardException with data set "failure 4"";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCard with data set "success 1"";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCard with data set "success 2"";d:0;s:84:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCard with data set "success 3"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 1"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 2"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 3"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 4"";d:0;s:7:"Warning";d:0.001;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 5"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 6"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testUpdateCardException with data set "failure 7"";d:0;s:82:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testRemoveCard with data set "success"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testRemoveCardException with data set "failure 1"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testRemoveCardException with data set "failure 2"";d:0;s:93:"Tests\CardsAppTest\ModelTest\CardsDeckTest::testRemoveCardException with data set "failure 3"";d:0;}}}